requirements.txt

```diff
@@ -43,6 +45,7 @@ future==0.15.2
 gensim==0.12.3
 google-api-python-client==1.5.1
 greenlet==0.4.10
+hiredis==0.2.0
```

Comment:
Ah no real reason why this is here, but I found it while working on other stuff. Free gift :) Optimizes redis access.